Gruntfile.js

```diff
@@ -25,6 +26,16 @@ module.exports = function (grunt) {
         grunt.log.writeln("Wrote api data to " + outputFile);
     };
 
+    var apiServer = require("./build/api-gen/dynamic-server.js");
+    function runApiServer() {
+      var done = this.async();
+      apiServer(grunt, "./build/api.json");
+      setTimeout(function(){
+        open("http://localhost:8080");
```

Comment:
Ah, I forgot to remove the grunt task.  It didn't quite work the way I wanted, and when I looked at the source I realized it was just a wrapper for open!